### DISCUSSION TITLE
Any option to disbale this on console.log ?

### DISCUSSION THREAD

**Discussion by naveencom** [unpoly/unpoly#480] [Q&A] [answered]
![unpoly-log](https://user-images.githubusercontent.com/57630189/235727759-a5c4bc9d-5ff9-4ad1-a26f-170bfe6c2f27.png)

Is there an option to hide this on console.log ?

**Comment by adam12** [comment, marked as answer]
The banner can be disabled before starting Unpoly, by using this option:

```javascript
up.log.config.banner = false
```

But Unpoly needs to be told not to start at `<script>` load time as it's too late, but manually once configuration is done. If you're using Unpoly via a `<script>` tag, you can do this by:

1. Configuring Unpoly not to automatically boot by setting the `[up-boot="manual"]` attribute on the `<script>` tag that loads Unpoly.
2. Calling `up.boot()` after configuring.

I made a quick example here: https://glitch.com/edit/#!/cake-treasure-lettuce?path=scripts.js%3A3%3A54

**Reply by naveencom** [reply]
Thank you Adam!

**Comment by naveencom** [comment]
![5646444646](https://user-images.githubusercontent.com/57630189/235850752-277b8dae-33f1-4dbe-a040-73db781869d0.png)

I didn't need to Boot. I'm using a self-hosted script.

Am I doing it correctly ?

**Reply by adam12** [reply]
Modifying the Unpoly script isn't recommended, and you need to disable boot because of ordering. My instructions above work for self-hosted scripts too.

```html
<script src="/unpoly.min.js" up-boot="manual"></script>
<script>
  up.log.config.banner = false
  up.boot()
</script>
```

**Reply by naveencom** [reply]
Thanks a lot, Adam!
